test/remote/gateways/remote_mondido_test.rb

```diff
@@ -0,0 +1,1487 @@
+require 'test_helper'
+
+class RemoteMondidoTest < Test::Unit::TestCase
```

Comment:
Was an attempt to run remote tests without merchant specific account. Changed to use hard coded values, but need to pass the credentials for you using a private channel.